src/Qowaiv.Validation.DataAnnotations/AnnotatedModelValidator.cs

```diff
@@ -85,10 +85,19 @@ private void ValidateProperties(NestedValidationContext context)
     private void ValidateProperty(AnnotatedProperty property, NestedValidationContext context)
     {
         var model = context.Instance;
-        var value = property.GetValue(model);
-
         var propertyContext = context.ForProperty(property);
 
+        object? value;
+        try
+        {
+            value = property.GetValue(model);
+        }
+        catch
+        {
```

Comment:
Is inaccessibility the only error you can get here?

Reply:
It should.